### PR TITLE
refector

### PR DESCRIPTION
**Pull request by muhammadusmannaeem**
- [x] Added Feature to store teacher details in .json format in respective folders.
- [x] Updated firebase read/write

### CODE REVIEW

File: src/parser.py [hunk 1]
```diff
@@ -78,21 +79,77 @@ def get_program(html) :
...
+        name_temp = str(html.find("div", {"class":"personal_info_full_name"}))
+        name = name_temp[37:(len(name_temp)-6)]
+        designation_temp = str(html.find("div", {"class":"personal_info_desig_title"}))
+        designation = designation_temp[39:(len(designation_temp)-6)]
+        level_temp = str(html.find("div", {"class":"personal_info_level_edu"}))
+        level = level_temp[37:(len(level_temp)-6)]
+        #email_temp = str(html.find("div", {"class" : "personal_info_email"}))
+        #email = email_temp[42:(len(email_temp)-26)]
+        email = "toBeAddedYet@haha.com"
+        data = "{\n'name': '" + str(name) + "',\n'designation': '" + str(designation) + "',\n'level': '" + str(level) + "',\n'email': '" + str(email) + "'\n}"
```
Comment (lablnet) [inline src/parser.py]: Make it Python Dict and then using `json` module to encode it

File: src/parser.py [hunk 2]
```diff
@@ -78,21 +79,77 @@ def get_program(html) :
...
+                    image = str(link)
+        name_temp = str(html.find("div", {"class":"personal_info_full_name"}))
+        name = name_temp[37:(len(name_temp)-6)]
+        designation_temp = str(html.find("div", {"class":"personal_info_desig_title"}))
+        designation = designation_temp[39:(len(designation_temp)-6)]
+        level_temp = str(html.find("div", {"class":"personal_info_level_edu"}))
+        level = level_temp[37:(len(level_temp)-6)]
+        #email_temp = str(html.find("div", {"class" : "personal_info_email"}))
+        #email = email_temp[42:(len(email_temp)-26)]
+        email = "toBeAddedYet@haha.com"
```
Comment (lablnet) [inline src/parser.py]: remove it with `email = ""`

**Review by lablnet — CHANGES_REQUESTED** [review]

**Comment by Usman-Naeem** [comment]
All the requested changes have been made.

**Comment by muhammadusmannaeem** [comment]
- [x] Requested Changes Completed
- [ ] Sorting Faculty details with names in general.py

**Review by lablnet — APPROVED** [review]

**Pull request merged**